src/pybind/mgr/cephadm/module.py

```diff
@@ -632,7 +638,7 @@ def _reconfig_ssh(self):
             self._ssh_options = None
 
         if self.mode == 'root':
-            self.ssh_user = 'root'
+            self.ssh_user = self.get_module_option('ssh_user')
         elif self.mode == 'cephadm-package':
             self.ssh_user = 'cephadm'
```

Comment:
In general, I can see two ways forward for the `package` mode: 

1. we're going to deprecate+remove it.
2. This line will get eventually be changed to:

    ```python
    self.ssh_user = self.get_module_option('ssh_user')
    ```
    as well

Can you please make a tracker issue that  the package mode has a hardcoded ssh user `cephadm`?

Reply:
https://tracker.ceph.com/issues/46097